src/main/java/entities/PostableItem.java

```diff
@@ -8,7 +8,7 @@
     protected String id;
 
     // The ID of the User that is the author of the PostableItem
-    private String authorId;
+    private final String authorId;
```

Comment:
I was wondering if we can get the author's username from a PostableItem class instead of just the author's id, especially when we are printing out the comments' users and the posts' creators.

Reply:
do you mean for comment? I guess you can write a method for it in PostManager